src/atc/delta/autoloader_handle.py

```diff
@@ -0,0 +1,240 @@
+from typing import List
+
+from pyspark.sql import DataFrame
+from pyspark.sql.streaming import DataStreamWriter
+
+from atc.configurator.configurator import Configurator
+from atc.functions import init_dbutils
+from atc.spark import Spark
+from atc.tables.SparkHandle import SparkHandle
+from atc.utils import GetMergeStatement
+from atc.utils.CheckDfMerge import CheckDfMerge
+from atc.utils.FileExists import file_exists
+
+
+class AutoLoaderHandle(SparkHandle):
+    def __init__(
+        self,
+        *,
+        name: str,
+        checkpoint_path: str,
+        location: str = None,
+        data_format: str = "delta",
+        # trigger_type ?
+    ):
+        super().__init__(name, location, data_format)
+
+        self._checkpoint_path = checkpoint_path
```

Comment:
you never check if this is empty. If it strictly required, raise an exception if it does not exist. A lot of code below will break if the path does not exist or is not writable.